readthedocs/templates/search/elastic_search.html

```diff
@@ -127,14 +127,16 @@ <h3>{% blocktrans with query=query|default:"" %}Results for {{ query }}{% endblo
                             {% if result.name %}
 
                               {# Project #}
-                              <a href="{{ result.url }}">{{ result.name }}</a>
+                              <a href="{{ result.url }}">{{ result.name }} (<em>{{ result.slug }}</em>)</a>
```

Comment:
If this is the notation for subproject results, perhaps we should mention "(from project <slug>)" or "(from project <name>)" would be even better. I don't have an example of how this looks with subproject search in doc right now though. If in-doc we omit "from project", we can omit here too.

Reply:
Huh, GitHub just allows HTML through apparently. <em>Test</em> <script>alert(0);</script>

Reply:
It's mostly because project's can have different names and Slugs, which can be confusing. Eg. in prod we have like 5000 projects with the name "Docs" or similar, so this helps make it explicit.

Reply:
I remember that we had a bug that makes all the results to say `(from project blah)` but I'm not sure what was the mistake. I think it was a problem with JS and the `===` instead of `==`. It may worth to check the history in case it's related to this.